CMakeLists.txt

```diff
@@ -508,10 +488,10 @@ install(
 install ( EXPORT hip-targets  FILE hip-targets.cmake NAMESPACE hip:: DESTINATION lib/cmake/hip  COMPONENT binary)
 
 # Generate .hipInfo
-file(WRITE "${PROJECT_BINARY_DIR}/lib/.hipInfo_install" ${_hipInfo_install})
-file(WRITE "${PROJECT_BINARY_DIR}/lib/.hipInfo" ${_hipInfo_build})
-install(FILES ${PROJECT_BINARY_DIR}/lib/.hipInfo_install
-  DESTINATION ${LIB_INSTALL_DIR} RENAME .hipInfo)
+file(WRITE "${PROJECT_BINARY_DIR}/bin/.hipInfo_install" ${_hipInfo_install})
+file(WRITE "${PROJECT_BINARY_DIR}/bin/.hipInfo" ${_hipInfo_build})
+install(FILES ${PROJECT_BINARY_DIR}/bin/.hipInfo_install
+  DESTINATION ${BIN_INSTALL_DIR} RENAME .hipInfo)
```

Comment:
.hipInfo should be placed under <install-dir>/lib/. HIPAMD install their hipInfo in there and hippcc reads the file from it.

Reply:
How do we know there are no other users that rely on the on the .hipInfo to be found under lib/ directory?

Reply:
.hipInfo is only used by hipcc and hipconfig where I made the adjustment as well. As soon as we merge this Release, I will  merge the HIPCC branch which makes the switch from Perl based hipcc to C++ based hipcc. The C++ hipcc does not use the .hipInfo for AMD/NVIDIA paths so we are free to put it anywhere we want and it makes more sense to have it in the same dir the as the hipcc executable.

Reply:
If you meant that the .hipInfo has been private to the hipcc and hipconfig then it’s fine to relocate the file.

> ... it makes more sense to have it in the same dir the as the hipcc executable.

Usually regular files are not installed under bin/ and lib/. Perhaps, its place is under share/?

Reply:
> Perhaps, its place is under share/?

Agree.